edge-middleware/feature-flag-split/app/page.tsx

```diff
@@ -29,22 +25,28 @@ export default function Index() {
         variant has a 50% chance)
       </Text>
       <div>
-        <Button
-          variant="secondary"
-          className="mr-2.5"
-          onClick={() => removeCookie(`flag-about`)}
-        >
-          Remove /about cookie & reload
-        </Button>
-        <Button
-          variant="secondary"
-          onClick={() => removeCookie(`flag-marketing`)}
-        >
-          Remove /marketing cookie & reload
-        </Button>
+        <CookieButton userKey="Joe" />
+        <CookieButton userKey="Bobby" />
       </div>
     </Page>
   )
 }
 
-Index.Layout = Layout
+function CookieButton({ userKey }: { userKey: string }) {
+  'use client'
```

Comment:
Is this needed given that we have 'use client' at the top?

Reply:
Probably not, seems like a left over from moving things around a few times